process/app.py

```diff
@@ -143,6 +143,8 @@ def process_message(tts_domain, tts_api_key, post_data):
                 snr = gateway['snr']
                 if snr < 0:
                     signal = rssi + snr
+            else:
+                snr = 0
```

Comment:
The scope of the `snr` variable was and still is incorrect. This declaration should rather be moved to above the `if`. 

e.g:
```
            rssi = gateway['rssi']
            snr = 0
            signal = rssi
            if 'snr' in gateway:
                snr = gateway['snr']
                if snr < 0:
                    signal = rssi + snr
```